### PR TITLE
Warn when manually deployed to production

### PR DESCRIPTION
**Pull request by mnishamk-freiheit**
When application is manually deployed to production, check that all its upstream environments are already updated, if not give warning. 

image

story: [SRX-5VUIF9](https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/-M37cZx7XYoOvQ5Mmcir/STORY/-N3YKNILyrB8A0ik5B41/)

### CODE REVIEW

**Review by hannesg — APPROVED** [review]
Makes sense!
I think we should have a follow up to only show the confirmation dialog when there is an actual warning.

**Comment by mnishamk-freiheit** [comment]
> Makes sense! I think we should have a follow up to only show the confirmation dialog when there is an actual warning.

Isn't that already present? as in if there are no locks or errors, it just adds it without a confirmation dialog

**Comment by hannesg** [comment]
> > Makes sense! I think we should have a follow up to only show the confirmation dialog when there is an actual warning.
> 
> Isn't that already present? as in if there are no locks or errors, it just adds it without a confirmation dialog

Ah, you are right. I'm still used to version 0.4.17

File: services/frontend-service/src/ui/ReleaseDialog.tsx [hunk 1]
```diff
@@ -343,6 +343,21 @@ export const ReleaseLockButton = (props: {
     );
 };
 
+const getUndeployedUpstream = (
```
[the remark this answers is not in the document]
Comment (mnishamk-freiheit) [reply services/frontend-service/src/ui/ReleaseDialog.tsx]: added

**Pull request merged**